Kudu.Services/Diagnostics/ProcessController.cs

```diff
@@ -456,6 +456,12 @@ private ProcessInfo GetProcessInfo(Process process, string href, bool details =
                 SetEnvironmentInfo(info);
             }
 
+            if (ArmUtils.IsArmRequest(Request))
+            {
+                info.EnvironmentVariables = null;
+                info.CommandLine = null;
+            }
```

Comment:
we could have populating these as a conditional above, but then we will lose the info for 

```csharp
processInfo.IsScmSite 
processInfo.IsWebJob 
processInfo.Description
```

which are deduced from the environment variables.